internal/user/storage/loader/configs.go

```diff
@@ -0,0 +1,54 @@
+package loader
+
+import (
+	"boilerplate/internal/cache"
+	"boilerplate/internal/user/storage"
+	"context"
+	"github.com/google/uuid"
+	"time"
+)
+
+var userFinder *storage.Queries
+
+func NewUserLoaderCache() *cache.LoaderCache[uuid.UUID, storage.User] {
+	baseCache := cache.NewCache[uuid.UUID, storage.User](
+		&cache.Config{
+			MaxCachedItems: 10,
+			CacheEnabled:   true,
+			LifeTime:       time.Hour,
+		},
+	)
+	return cache.NewLoaderCache(baseCache)
+}
+
+func NewUserLoaderConfig(finder *storage.Queries, cache *cache.LoaderCache[uuid.UUID, storage.User]) UserLoaderConfig {
```

Comment:
I propose to avoid creating context-related loaders. So we will create a loader for the whole time the application is working. This approach is more performant than creating a loader per request. But on contrary, it has some drawbacks, and one of them is necessary for operating low-level data without transforming it into a response for the current user.